ansible/files/vmware_api.py

```diff
@@ -69,6 +70,21 @@ def get_vm_folder(self, folder_path):
         return self.content.searchIndex.FindByInventoryPath(inventory_path)
 
 
+    def select_destination_host_and_datastore(self):
+        datastores = self._get_datastores_from_cluster()
+        if len(datastores) == 0:
+            return None, None
+        random.shuffle(datastores)
+        chosen_datastore = datastores[0]
+        chosen_host = chosen_datastore.host[0].key
+        return chosen_host, chosen_datastore
+
+
+    def _get_datastores_from_cluster(self):
+        datastores = [d for d in self.datacenter.datastore if 'ssd' in d.name]
```

Comment:
Please see a comment regarding choosing cluster. Storage pod object (http://pubs.vmware.com/vsphere-6-5/index.jsp#com.vmware.wssdk.apiref.doc/vim.StoragePod.html) has references to its datastores.